.circleci/scripts/test-merchant-branches.sh

```diff
@@ -22,35 +20,19 @@ fi
 echo "The following branches will be tested after rebasing against $baseBranch"
 while IFS= read -r branchName || [[ -n "$branchName" ]]; do
   if [ ${#branchName} -gt 0 ]; then
-    merchantBranches+=("ci/$branchName")
-    echo "ci/$branchName"
-  fi
-done < "$configFile"
-
-# TODO: Add support to rebase and run tests for all merchant branches in the config
-# For now, we only rebase and run tests for the first merchant branch in the config
-branchName=${merchantBranches[0]}
-if ! (git checkout "$branchName"); then
-  echo "Failed to checkout branch $branchName"
-  exit 1
-fi
-if ! (git rebase origin/$baseBranch); then
-  echo "Failed to rebase branch $branchName on $baseBranch"
-  git rebase --abort
-  exit 1
-fi
-if [ "$isIntegration" = true ]; then
-  echo "Start integration tests..."
-  if ! ./Test/scripts/ci-integration.sh; then
-    echo "integration tests failed"
-    exit 1
-  fi
-else
-  echo "Start unit tests..."
-  if ! ./Test/scripts/ci-unit.sh; then
-    echo "unit tests failed"
-    exit 1
-  fi
-fi
+    merchantBranch="ci/$branchName"
+    echo "$merchantBranch"
 
+    curl -X POST \
```

Comment:
Can you print the response from this curl command? the output looks very sparse right now.

https://app.circleci.com/pipelines/github/BoltApp/bolt-magento2/3538/workflows/479fe8a3-e7ac-473c-b4f0-366bed1834b7/jobs/15885

Reply:
Also where do I see the output of `.circleci/scripts/rebase-and-unit-test.sh`

Reply:
The output should be on the job that gets triggered on the merchant branch

Reply:
> The output should be on the job that gets triggered on the merchant branch

I am not sure how to easily get to the triggered job. The output from curl I think should have a link to the triggered workfow?

Also it should tell us if the call failed and got a non-200 response. Currently I just see the following which is not helpful

<img width="926" alt="image" src="https://user-images.githubusercontent.com/61712645/88716381-fa64f100-d0ec-11ea-855f-7f408c605c26.png">